app/job/send_event_to_bigquery_job.rb

```diff
@@ -0,0 +1,10 @@
+class SendEventToBigqueryJob < ApplicationJob
+  TABLE_NAME = 'events'.freeze
```

Comment:
This will be going to a new events table that's specific to Find. This table has a slightly updated format and we'll likely back-port some changes to the Apply events table once things are more settled.